modules/configure-acs-kube-security-platform.adoc

```diff
@@ -0,0 +1,52 @@
+[id="configure-acs-kube-security-platform"]
+= Configuring the {product-title} platform
+:context: configure-acs-kube-security-platform
+
+You can create a new integration in the {product-title} platform by using the email plugin notifier.
+
+.Procedure
+
+. Navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Plugins* section, select *Email*.
```

Comment:
There is no plugins section on this page. I think you are looking under "Notifier Integrations"